### PR TITLE
changes - March 15 2024

### PR DESCRIPTION
**Pull request by amanueltsfy**
- course highlights implemented
- admin new course feature platform added

### CODE REVIEW

File: src/assets/locales/en.json [hunk 1]
```diff
@@ -9,23 +9,34 @@
       "the_course_has_no_lessons_yet": "The course has no lessons yet!",
       "back_to_enrolled_courses": "Back To Enrolled Courses",
       "youve_completed_the_course": "You've completed the course",
-      "we_could_not_find_any_courses. thanks for you patient": "We could not find any courses. thanks for you patient"
+      "we_could_not_find_any_courses. thanks for you patient": "We could not find any courses. thanks for you patient",
```
Comment (jeznag) [inline src/assets/locales/en.json:12]: When would this appear?

File: src/assets/locales/en.json [hunk 2]
```diff
@@ -9,23 +9,34 @@
...
+      "platform": "Platform"
     },
     "lesson": {
       "new_lesson": "New Lesson",
-      "no_lesson_found": "No lesson found, please add one.",
+      "no_lesson_found_add_one": "No lesson found, please add one.",
```
Comment (jeznag) [inline src/assets/locales/en.json:17]: Time to add the first lesson

File: src/assets/locales/en.json [hunk 3]
```diff
@@ -9,23 +9,34 @@
...
+      "no_lesson_found": "No lesson found",
       "youtube_url": "Youtube URL",
       "lesson_title": "Lesson {{position}}",
       "youve_not_enrolled_in_any_course_yet": "You've not enrolled in any course yet."
     },
     "error": {
       "oops_something_went_wrong": "Oops, something went wrong",
       "couldnt_able_to_create_course": "Couldn't able to create course.",
-      "couldnt_able_to_update_course": "Couldn't able to update course."
+      "couldnt_able_to_update_course": "Couldn't able to update course.",
```
Comment (jeznag) [inline src/assets/locales/en.json:26]: "Something went wrong while creating the course. Very sorry. Please email support@focusbear.io and tell us what led to this error"

Use same language for below

File: src/assets/locales/en.json (hunk 1 above)
Comment (amanueltsfy) [reply src/assets/locales/en.json:12]: image

**Pull request merged**